tests/python/unittest/test_gluon_data.py

```diff
@@ -205,24 +206,20 @@ def batchify(self, data):
                 nd.array(labels, dtype=labels.dtype, ctx=context.Context('cpu_shared', 0)),
                 nd.array(y_lens, ctx=context.Context('cpu_shared', 0)))
 
-
-    # This test is pointless on Windows because Windows doesn't fork
-    if platform.system() != 'Windows':
-        data = Dummy(True)
-        loader = DataLoader(data, batch_size=40, batchify_fn=data.batchify, num_workers=2)
-        for epoch in range(1):
-            for i, data in enumerate(loader):
-                if i % 100 == 0:
-                    print(data)
-                    print('{}:{}'.format(epoch, i))
-
-        data = Dummy(True)
-        loader = DataLoader(data, batch_size=40, batchify_fn=data.batchify_list, num_workers=2)
-        for epoch in range(1):
-            for i, data in enumerate(loader):
-                if i % 100 == 0:
-                    print(data)
-                    print('{}:{}'.format(epoch, i))
+    # sanity tests
+    # enable test on windows
+    data = Dummy(False)
```

Comment:
Did you intentionally set the randomized shape to false? It used to be true before.

Also, could elaborate on the comment in line 210? Is there anything specific we have to do on Windows or is it working out of the box? If it doesn't need any special configuration, we can safely remove the comment as it is otherwise confusing to people who don't know the history

Reply:
- I am deliberately switch one test case to fixed shape, and the rest one is responsible for the random test.
- Will remove the unnecessary comments

BTW, my PR disappeared in CI yesterday but show up today, do you have a clue?

Reply:
I see, thanks for elaborating.

Yeah, we had some disk space problems on our server. Everything should be fine now :)